openapi-codegen/core/src/test/scala/sttp/tapir/codegen/TestHelpers.scala

```diff
@@ -134,8 +134,9 @@ object TestHelpers {
         "/books/{genre}/{year}",
         Seq(
           OpenapiPathMethod(
-            methodType = "post",
+            methodType = "get",
```

Comment:
Extracting the `OpenapiPathMethod` via a traverse on legal names means that this seq is now given a canonical ordering by method name - hence flipping the post and get in this test. Since the underlying yaml is technically a map and not a list, I think that's fairly legit.

Reply:
if there's a fixed ordering, the test should be more resilient now, so this is fine of course